crates/driver/src/print_build.rs

```diff
@@ -927,15 +983,13 @@ pub async fn print_build(
         .into_iter()
         .filter(|(k, _v)| !k.starts_with('@'))
     {
-        let graph_node = graph_node.clone();
         let element = if !opt.no_aggregate_source {
             entry
         } else {
             to_directory(entry.to_string())
         };
         let v = graph_nodes.entry(element).or_default();
         v.push(graph_node);
-        v.sort_by(|a, b| a.node_label.cmp(&b.node_label))
```

Comment:
this is moved inside the printing function. It isn't great to call sort after adding each element. We should only do it once.